app/src/main/java/com/ucmobiledevelopment/freeways/MainViewModel.kt

```diff
@@ -72,7 +74,7 @@ class MainViewModel(var incidentService : IIncidentService = IncidentService())
             incident.incidentId = document.id
             val handle = document.set(incident)
             handle.addOnSuccessListener { Log.d("Firebase", "Document saved") }
-            handle.addOnFailureListener { Log.e("Firebase", "Save failed $it") }
+            handle.addOnFailureListener { Log.e("Firebase", "Save failed $user") }
```

Comment:
These are different variables:
it == the error message
user == a user

I'd keep this as it were, as the error message is the most important thing to see in a log.